### ISSUE TITLE
Linux only: Guides can be dragged off their axis

### ISSUE THREAD

**Issue by destroytoday** [destroytoday/Dwarf#2]
Dragging a guide should only drag it on its opposing axis, but in Linux, it drags like a normal window. This is an AIR bug that will be logged.

**Comment by bkuri** [comment]
I cannot reproduce this. If I drag a vertical line towards the bottom, I can the guide immediately snaps back to the top.

**Comment by destroytoday** [comment]
Right. If you have a chance to test on Windows or Mac, you'll see that there's no need to snap back because the guide is always confined to its opposite axis.

**Comment by bkuri** [comment]
OK I see what you mean. I had a chance to test this on Windows and it doesn't move at all. Linux bug it is.
